src/Microsoft.TemplateEngine.Orchestrator.RunnableProjects/SimpleConfigModel.cs

```diff
@@ -896,6 +866,59 @@ private List<FileSourceMatchInfo> EvaluateSources(IParameterSet parameters, IVar
             return sources;
         }
 
+        private void AugmentRenames(IFileSystemInfo configFile, ref string sourceTargetName, object resolvedNameParamValue, IParameterSet parameters, Dictionary<string, string> fileRenames)
+        {
+            List<KeyValuePair<string, string>> fileRenameMappings = new List<KeyValuePair<string, string>>();
+            Dictionary<string, string> coreRenames = new Dictionary<string, string>(fileRenames);
+
+            if (resolvedNameParamValue != null && SourceName != null)
+            {
+                string targetName = ((string)resolvedNameParamValue).Trim();
+                sourceTargetName = sourceTargetName.Replace(SourceName, targetName);
+                fileRenameMappings.Add(new KeyValuePair<string, string>(SourceName, targetName));
+            }
+
+            foreach (IExtendedTemplateParameter p in parameters.ParameterDefinitions.OfType<IExtendedTemplateParameter>())
+            {
+                if (!string.IsNullOrEmpty(p.FileRename))
+                {
+                    if (parameters.TryGetRuntimeValue(EnvironmentSettings, p.Name, out object value) && value is string s)
+                    {
+                        fileRenameMappings.Add(new KeyValuePair<string, string>(p.FileRename, s));
+                    }
+                }
+            }
+
+            foreach (KeyValuePair<string, string> entry in coreRenames)
+            {
+                foreach (KeyValuePair<string, string> rename in fileRenameMappings)
+                {
+                    string outRel = entry.Value.Replace(rename.Key, rename.Value);
```

Comment:
Yep, these are the "output relative path" and "template relative path" values for a particular piece of content. I'll rename them